plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,14 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important notes:
+
+* Pulpcore installs a separate PostgreSQL database. Databases should be upgraded to avoid conflicts.
```

Comment:
As a user I'm not sure what `Databases should be upgraded to avoid conflicts.` means if it follows `Pulpcore installs a separate PostgreSQL database`. Upgraded to what?